packages/portal/nuxt.config.js

```diff
@@ -18,6 +18,7 @@ import features, { featureIsEnabled, featureNotificationExpiration } from './src
 import { BASE_URL as EUROPEANA_ENTITY_API_BASE_URL } from './src/plugins/europeana/entity.js';
 import { BASE_URL as EUROPEANA_ENTITY_MANAGEMENT_API_BASE_URL } from './src/plugins/europeana/entity-management.js';
 import { BASE_URL as EUROPEANA_RECORD_API_BASE_URL } from './src/plugins/europeana/record.js';
+import { FULLTEXT_BASE_URL as EUROPEANA_RECORD_API_FULLTEXT_URL } from './src/plugins/europeana/record.js';
```

Comment:
this is highlighted as a codesmell due to importing from record.js twice

Reply:
Indeed. imports combined.